tests/cpu_bugs/ipmi_to_qemu.pm

```diff
@@ -27,28 +27,33 @@ use ipmi_backend_utils;
 use power_action_utils 'power_action';
 use testapi;
 use utils;
-use version_utils 'is_sle';
+use version_utils 'is_sle', 'get_sles_release';
```

Comment:
If is_sle function doesn't be used, I think it can be removed?

Reply:
Thanks, removed.